src/config/constants.js

```diff
@@ -11,6 +11,7 @@ export const PD_OAUTH_CLIENT_ID = env.REACT_APP_PD_OAUTH_CLIENT_ID || null;
 export const PD_OAUTH_CLIENT_SECRET = env.REACT_APP_PD_OAUTH_CLIENT_SECRET || null;
 export const PD_SUBDOMAIN_ALLOW_LIST = env.REACT_APP_PD_SUBDOMAIN_ALLOW_LIST || '*';
 export const PD_USER_TOKEN = env.REACT_APP_PD_USER_TOKEN || null;
+export const PD_REQUIRED_ABILITY = env.REACT_APP_PD_REQUIRED_ABILITY || null;
```

Comment:
Say the environment variable `REACT_APP_PD_REQUIRED_ABILITY` was not configured, the constant `PD_REQUIRED_ABILITY` would be `null` - what would happen when we check the abilities?

```javascript
!abilities.includes(PD_REQUIRED_ABILITY)
```

Reply:
Thanks @gsreynolds.

If `REACT_APP_PD_REQUIRED_ABILITY` is not set (and `PD_REQUIRED_ABILITY` == null), the same intended logic will apply.

i.e. users will not be able to use the `log_entries` polling capability.